src/App.js

```diff
@@ -1,17 +1,31 @@
 import React from 'react';
 import './App.css';
-import Calculater from './Components/Calculater';
+import { Route, Switch } from 'react-router-dom';
+import Calculator from './pages/Calculator';
+import Home from './pages/Home';
+import Quote from './pages/Quote';
+import Navbar from './pages/Navbar';
 
 class App extends React.PureComponent {
   render() {
     return (
-      <div className="App">
-        <h1>
-          Hello from App
-        </h1>
-        <Calculater />
+      <>
+        <Navbar />
+        <Switch>
+          <Route exact path="/">
+            <div className="App">
+              <Home />
 
-      </div>
+            </div>
+          </Route>
+          <Route exact path="/calculator">
+            <Calculator />
```

Comment:
- [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md) for this project, " `The general layout should match these wireframes:` ", Please make sure you Calculator page matches the [wireframe](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md), the calculator itself occupies about 2/3 of page and the 
 text (`Let's do some math`) is aligned with the page title (`Math Magicians`). Please kindly fix that.

Reply:
Sure, thanks for your feedback